PyEveLiveDPS/labelHandler.py

```diff
@@ -124,27 +182,28 @@ def enableTotal(self, findColor, enable=True):
 
     def convertNumberToStr(self, number):
         decimals = int(self.decimalPlaces)
-        formatString = '{:,.'+str(decimals)+'f}'
+        formatString = '{:,.' + str(decimals) + 'f}'

```

Comment:
Note: We may want to i18n displaying numbers in the future as well.  Not needed for this PR I think.

Reply:
OK.